config.docker.yml

```diff
@@ -6,4 +6,5 @@ ALGOLIA_INDEX_PREFIX: 'ALGOLIA_INDEX_PREFIX_REPLACE_ME'
 ALGOLIA_APPLICATION_ID: 'ALGOLIA_APPLICATION_ID_REPLACE_ME'
 ALGOLIA_READ_ONLY_API_KEY: 'ALGOLIA_READ_ONLY_API_KEY_REPLACE_ME'
 
-MOHCD_SUBDOMAIN: 'MOHCD_SUBDOMAIN_REPLACE_ME'
+# this is not sensitive data, so we just pass it in as is
+MOHCD_SUBDOMAIN: 'sfserviceguide'
```

Comment:
Actually, this shouldn't change. The magic string needs to be there so that the Docker container is able to substitute the value out at container runtime. This allows us to change the value without having to do a rebuild and push of the Docker image. I'm not sure if the MOHCD subdomain has been finalized yet, so I think there's still utility in keeping it a parameter.

Reply:
@richardxia In that case, what is the right way to go about this?

Reply:
I'd say that we should revert the changes back to what they were and make sure that the Docker containers in staging/prod have the right value set. I think we're keeping our container settings in https://github.com/ShelterTechSF/gcloud, but @lgarofalo can correct me if I'm wrong.

Reply:
Ok – I have reverted the change `config.docker.yml`. I have created a PR into the gcloud repo, and requested that you and/or Leland review it: https://github.com/ShelterTechSF/gcloud/pull/6

Reply:
@richardxia – I chatted with @lgarofalo and he advised that we add a new config value, MOHCD_DOMAIN, to the container settings. He's applied my PR here: https://github.com/ShelterTechSF/gcloud/pull/6

I've added the MOHCD_DOMAIN config to the relevant files in this repo. Please let me know if I'm missing something. Thank you!